client/src/components/Employees/Employees.css

```diff
@@ -0,0 +1,19 @@
+.topWordss{
+    margin-top: 2%;
+    font-family: 'Open Sans', sans-serif;
+    display: flex;
+    justify-content: space-between;
+}
+.header{
+    font-size: 25px;
+    padding-left: 8%;
+    float: left;
+    color: black;
+}
+.add{
+    font-size: 18px;
+    text-align: right;
+    float: right;
+    padding-right: 10%;
+    color: #3626A7;
+}
```

Comment:
Let's end all files with a newline (that's what the red cross thing means). Ik it seems minor but they ask you to do this stuff in industry too.